mithril-aggregator/src/http_server/routes/proof_routes.rs

```diff
@@ -98,14 +89,35 @@ mod handlers {
             }
         }
     }
+
+    pub async fn build_response_message(
```

Comment:
IMO the `ToCardanoTransactionsProofsMessageAdapter::try_adapt` should take the `signed_entity_type` as argument and be responsible for verifying that the message is well formed. This `build-response_message` function does not seem to be needed in that case.